Header header
float64[] gain_bias_err